postgres-appliance/build_scripts/base.sh

```diff
@@ -146,6 +146,22 @@ for version in $DEB_PG_SUPPORTED_VERSIONS; do
         done
     )
 
+    if [ "${TIMESCALEDB_APACHE_ONLY}" != "true" ] && [ "${TIMESCALEDB_TOOLKIT}" = "true" ]; then
+        __versionCodename=$(sed </etc/os-release -ne 's/^VERSION_CODENAME=//p')
+        echo "deb [signed-by=/usr/share/keyrings/timescale_E7391C94080429FF.gpg] https://packagecloud.io/timescale/timescaledb/ubuntu/ ${__versionCodename} main" | tee /etc/apt/sources.list.d/timescaledb.list
+        curl -L https://packagecloud.io/timescale/timescaledb/gpgkey | apt-key --keyring /usr/share/keyrings/timescale_E7391C94080429FF.gpg add -
+
+        apt-get update
+        if [ $(apt-cache search --names-only "^timescaledb-toolkit-postgresql-${version}$" | wc -l) -eq 1 ]; then
+            apt-get install "timescaledb-toolkit-postgresql-$version"
```

Comment:
Btw; this is a problem re. reproducible builds, in that the newest version for a given pgversion always will be selected. Is this a show-stopper?

I'm not quite what to do about this if it is, as it seems the (not so good, frankly) alternative is having an array of what exact toolkit version to install for what pgversion. What's your thoughts on this?

Reply:
unfortunately, Spilo does not guarantee reproducible builds. it is reprodicible only when is pulled from a registry:)

Reply:
Yeah, true enough. What I mean is more that this dependency is probably likely to receive quite a bit more updates than most others and I'd imagine a high likelyhood of version changes for plugins on different builds of the same spilo release isn't great.

If you don't think it's a problem, I'm fine with keeping it as is, but if you think it's a problem I can add version fixing.